### PR TITLE
Updated GitHub teams for localization teams

### PR DESCRIPTION
**Pull request by rlenferink**
/cc @zacharysarah 
/sig docs

### CODE REVIEW

File: sig-docs/README.md [hunk 1]
```diff
@@ -54,9 +54,13 @@ Note that the links to display team membership will only work if you are a membe
 | --------- |:-------:| ----------- |
 | @kubernetes/sig-docs-maintainers | [link](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers) | Documentation maintainers |
 | @kubernetes/sig-docs-pr-reviews | [link](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews) | Documentation PR reviews |
-| @kubernetes/sig-docs-ko-owners | [link](https://github.com/orgs/kubernetes/teams/sig-docs-ko-owners) | Korean localization |
-| @kubernetes/sig-docs-ja-owners | [link](https://github.com/orgs/kubernetes/teams/sig-docs-ja-owners) | Japanese localization |
-| @kubernetes/sig-docs-zh-owners | [link](https://github.com/orgs/kubernetes/teams/sig-docs-zh-owners) | Chinese localization |
+| @kubernetes/sig-docs-en-owners | [link](https://github.com/orgs/kubernetes/teams/sig-docs-en-owners) | English content (default) |
```
Comment (nikhita) [inline sig-docs/README.md]: This needs to be updated in sigs.yaml.

Comment (rlenferink) [reply sig-docs/README.md]: My bad. Just updated sigs.yaml as well ;)

**Comment by zacharysarah** [comment]
@rlenferink Ping, looks like CI is failing here.

**Comment by rlenferink** [comment]
Thanks @zacharysarah it indeed was because I completely forget to update the sigs.yaml instead of the README. I've updated the sigs.yaml and CI is succeeding now 😃

**Comment by nikhita** [comment]
/assign @idvoretskyi 
for approval

**Pull request merged**